src/v2/Components/NavBar/Menus/MobileNavMenu/MobileNavMenu.tsx

```diff
@@ -261,29 +260,50 @@ const AuthenticateLinks: React.FC = () => {
   )
 }
 
-export const LoggedInLinks: React.FC<any> = ({ account }) => {
+export const LoggedInLinks: React.FC<any> = () => {
   const { mediator, user } = useSystemContext()
+  let menu = {
+    title: "Account",
+    links: [
```

Comment:
Exactly: since the desktop implementation is already bypassing the JSON definition (and has slightly different structure), it seemed like a pointless complication to put the mobile entries there